vite.config.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  build: {
+    outDir: '_site'
+  }
+}
```

Comment:
Looks like Prettier isn't being run on this file, since `eslint` is only running against `src`. Could change `test:lint` script to run `eslint .`, though might need to ignore a few paths with an `.eslintignore`.